lib/aerospike/query/pred_exp/regex.rb

```diff
@@ -0,0 +1,50 @@
+# fr# frozen_string_literal: true
+
+module Aerospike
+  class PredExp
+    class Regex < PredExp
+      def initialize(op, flag = Flags::NONE)
+        @op = op
+        @flag = flag
+      end
+
+      def estimate_size
+        10
+      end
+
+      def write(buffer, offset)
+        # write op type
+        buffer.write_int16(@op, offset)
+        offset += 2
+
+        # write length
+        buffer.write_int32(4, offset)
+        offset += 4
+
+        # write predicate count
+        buffer.write_int32(@flag, offset)
+        offset += 4
+
+        offset
+      end
+
+      # Regex bit flag
+      module Flags
```

Comment:
This seems a duplicate of `Aerospike::PredExp::RegexFlags`. Do we really need this module?